### PR TITLE
Implement homework JS2 W3

### PR DESCRIPTION
**Pull request by valmdz**

### CODE REVIEW

File: javascript/javascript2/week3/indexWarmup.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+ <meta charset="UTF-8">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <!-- <script defer src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY&callback=initMap"></script> -->
```
Comment (piyushpalod) [inline javascript/javascript2/week3/indexWarmup.html:7]: Tip : always remove the code which is commented and not in use , when raise final PR.

File: javascript/javascript2/week3/script.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  countL = 0;
+  textS.hidden = false;
+  textL.hidden = false;
+  resultS.hidden = true;
+  resultL.hidden = true;
+};
+
+const startGame = () => {
+  console.log(timeLimit.value);
+  const displayWinner = () => {
```
Comment (piyushpalod) [inline javascript/javascript2/week3/script.js:25]: function inside function doesn't make sense. you can have implementation of dispalyWinner() function outside of startGaame() and then you can call it inside.

**Review by piyushpalod — APPROVED** [review]
Nice work !! keep it up.

File: javascript/javascript2/week3/indexWarmup.html (hunk 1 above)
Comment (valmdz) [reply javascript/javascript2/week3/indexWarmup.html:7]: ok, thanks

File: javascript/javascript2/week3/script.js (hunk 2 above)
Comment (valmdz) [reply javascript/javascript2/week3/script.js:25]: cool

**Pull request merged**